test/source-build-check.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Could we add a note for users that this is intended to verify `MANIFEST.in` and is not of much interest to end users?